set.mm

```diff
@@ -53537,6 +53554,29 @@ of the relation (see ~ op1stb ).  (Contributed by NM, 28-Jul-2004.) $)
       NRABGOABCDEHFPQ $.
   $}
 
+  ${
+    $d D x $.  $d x A $.  $d x C $.  $d x ph $.
+    elrnmptspd.1 $e |- F = ( x e. A |-> B ) $.
+    elrnmptspd.2 $e |- ( ph -> C e. A ) $.
+    elrnmptspd.3 $e |- ( ph -> D e. V ) $.
+    elrnmptspd.4 $e |- ( ( ph /\ x = C ) -> D = B ) $.
+    $( Elementhood in an image set.  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
+    elrnmptspd $p |- ( ph -> D e. ran F ) $=
```

Comment:
This theorem should be called ~elrnmptd, but this label is already used in GS's mathbox. So either the theorem in GS's mathbox should be changed, or we change ~elrnmptspd to ~elrnmptdv (according to our conventions: "Additionally, we sometimes suffix with "v" the label of a theorem adding a disjoint variable condition").

Furthermore, the term "image set" is not specified appropriately (although it is used often): According to ~imadmrn, "**the** image set" of a function means "the image of the domain" of a function, and is synonymous with "range". But `ran F` is not always a set (it can be a proper class), therefore "**the** image" of a function would be a better synonym for "range".

Reply:
I agree. But looking at the comment of ~ elrnmpt, the comment should be "Elementhood in the range of a function in maps-to notation, deduction form."

Reply:
Named it ` elrnmptdv ` and used your comment suggestion.